packages/estatico-boilerplate/.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+
+[package.json]
```

Comment:
@orioltf, I think we can drop this part since it does not differ from the default above